pkg/csi_driver/controller.go

```diff
@@ -736,9 +736,7 @@ func (s *controllerServer) CreateSnapshot(ctx context.Context, req *csi.CreateSn
 	backupObj, err := s.config.fileService.CreateBackup(ctx, filer, req.Name, util.GetBackupLocation(req.GetParameters()))
 	if err != nil {
 		glog.Errorf("Create snapshot for volume Id %s failed: %v", volumeID, err)
-		if file.IsUserError(err) {
-			return nil, status.Error(codes.InvalidArgument, err.Error())
-		} else {
+		if err != nil {
```

Comment:
not needed?

Reply:
Did you see [comment](https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/pull/249#issuecomment-1095600739). I wasn't able to reproduce this error. When I tried I got `Failed to create snapshot content with error failed to get input parameters to create snapshot backup-source-pvc: "the PVC source-pvc is not yet bound to a PV, will not attempt to take a snapshot"`